bufferserver/src/main/java/com/datatorrent/bufferserver/packet/MessageType.java

```diff
@@ -38,6 +38,7 @@
   public static final byte RESET_REQUEST_VALUE = 9;
   public static final byte CHECKPOINT_VALUE = 10;
   public static final byte CODEC_STATE_VALUE = 11;
+  public static final byte MODIFY_CONFIGURATION_VALUE = 12;
```

Comment:
I already have these in the local file. Could you backout the changes in this file only.